internal/clearingway/handlers.go

```diff
@@ -325,6 +326,13 @@ func (c *Clearingway) InteractionCreate(s *discordgo.Session, i *discordgo.Inter
 				c.Uncolor(s, i)
 			case CommandRemoveAll:
 				c.RemoveAll(s, i)
+			case CommandRemoveEncounter:
+				if ok := len(command) > 2; !ok {
+					fmt.Printf("Invalid custom ID received: \"%v\"\n", customID)
+					return
+				}
+				// can reuse this function since button interactions have no selections
```

Comment:
i would prob add a tag like  `// NB: can reuse...`. If you're curious NB is basically explaining a little hack, something that's not really optimal, or something that's not really obvious to the reader